docs/Cluster-VictoriaMetrics.md

```diff
@@ -273,6 +273,14 @@ See [trobuleshooting docs](https://docs.victoriametrics.com/Troubleshooting.html
   Snapshots may be created independently on each `vmstorage` node. There is no need in synchronizing snapshots' creation
   across `vmstorage` nodes.
 
+## Tenant routing with labels
+
+ It's possible to define tenant id with metric labels. Meta labels - `vm_project_id` and `vm_account_id` are used for this case. Such labels must have numeric value. 
+For instance, `my_metric_name{vm_project_id="5",vm_account_id="15",job="special"}` will be routed to the `15:5` tenant.
+ Meta labels will be stripped and result metric will be `my_metric_name{job="special"}`. It's also possible to use `extra_label` or metric `relabeling` for tenant routing.
+
+This functional could be enabled with `vminsert` flag `-parseTenantIDFromLabels=true`. By default, it's disabled for security purpose.
+
```

Comment:
```suggestion
### Via meta labels

_Multitenancy via meta labels can be enabled by passing flag `-parseTenantIDFromLabels=true` to `vminsert` component. By default, it's disabled for security reasons._

When URL-based multitenancy isn't an option, try using meta labels `vm_project_id` and `vm_account_id` attached to time series.

For example, `my_metric{vm_project_id="5",vm_account_id="15"}` is parsed and routed by vminsert component to the tenant `15:5`. Meta labels are stripped from the original time series once written to the vmstorage.

One write request or scrape target can contain time series with different tenants. It's also possible to use `extra_label` or [relabeling](https://docs.victoriametrics.com/#relabeling) for modifying or adding multitenancy meta labels.

Prefer meta labels for multitenancy in cases when you need to have hundreds and thousands of tenants.
```

I have few questions it would be great to answer here:
* what happens if payload with meta labels is sent to specific tenant's URL?
* what happens if project or account id labels are omitted?

Reply:
It's valid questions and we have to add documentation for it.

 Specific tenant's URL contains default tenant, which will be used if project or account id labels are omitted.

 It's not possible to send requests without tenant id specification.